include/hstcalerr.h

```diff
@@ -45,8 +79,16 @@
 #define COPY_NOT_POSSIBLE      1025
 
 #define INVALID_VALUE          1111
-#define GENERIC_ERROR_CODE     1111 //used in STIS
+#define GENERIC_ERROR_CODE     1112 //used in STIS
```

Comment:
I don't know about changing this now... but then again I don't think anyone memorized the error codes by number even after 35 years, so... 🤷‍♀️

Reply:
Yeah... Hey, at least the status and the messages are unique now.

Reply:
Does DMS actually check the actual int value of individual err status? I am just a tiny bit worried this is going to break a status check but maybe not. 🎲

Reply:
Since this is STIS, let's ask @stscirij

Reply:
From what I can tell, there are no references to INVALID_VALUE.  The only reference to 1111 is here, in a comment:
`lib/cs1/getkeyinfo1.c:	Change status values 1111 and 1003 to GENERIC_ERROR_CODE.`
There are many places where GENERIC_ERROR_CODE is returned, and many comments stating that a particular error code was replaced by GENERIC_ERROR_CODE (probably a dozen or so different numbers in the 1000-1100 range).  I don't see any references to the value 1112 in the stis code or headers, so I think it should be OK to remap GENERIC_ERROR_CODE to 1112.

Reply:
Thanks, Robert! Is it slightly safer than to remap INVALID_VALUE instead?

Reply:
I don't think it matters - unless one of the other instruments uses 1111 or INVALID_VALUE, we don't need it, so the line setting INVALID_VALUE to 1111 could just go and we could keep GENERIC_ERROR_CODE at 1111.  But changing GENERIC_ERROR_CODE to 1112 should also work and allow use of INVALID_VALUE if needed.

Reply:
> Does DMS actually check the actual int value of individual err status? I am just a tiny bit worried this is going to break a status check but maybe not. 🎲

I don't think DMS parses the trailer file for status information. As for exit codes, grep says we're only using the following:
- `0`
- `1`
- `ERROR_RETURN` (2)
- `OUT_OF_MEMORY` (111)

There is one instance of `exit(status)` in `cs12.c`, but it only leads to a function that sets `status` to one of the values above.